packages/components/src/components/domainproperties/entities/EntityDetailsForm.tsx

```diff
@@ -20,6 +20,9 @@ interface Props {
     nameExpressionPlaceholder?: string
 }
 
+//TODO rename above
+export type EntityDetailsProps = Props;
```

Comment:
I agree with the TODO, can you just rename the above Props var?

Reply:
fixed.